### DISCUSSION TITLE
pull lazer songs from %osu!directory%\Songs

### DISCUSSION THREAD

**Discussion by calebcalamity** [ppy/osu#30954] [Ideas] [answered]
lazer takes a long time to import maps, and you can't edit them so, just don't use encryption and pull songs from the osu stable directory

**Comment by bdach** [comment, marked as answer]
There is no "encryption". And you can modify beatmaps directly via the editor (there's an option for external editing).

We're not going back to the old structure.

**Comment by peppy** [comment]
Also just to clarify, the time taking to import is likely due to retrieval of metadata to make the beatmap work better in the game. This will be optimised in the future.

**Reply by calebcalamity** [reply]
ah
